### PR TITLE
Wazuh release v3.11.1 7.5.1

### LINKED ISSUE #81: Wazuh Release 3.11.1_7.5.1
Wazuh version: 3.11.1
Elastic version: 7.5.1

### Tasks

- [x] Adapt to new versions (3.11.1 - 7.5.1)

- [x] Check templates and update them if needed

- [x] Tests

- [x] Update changelog

- [ ] Tag: v3.11.1_7.5.1

- [ ] Update documentation references

- [ ] Draft release

### PR DESCRIPTION
**Pull request by rshad**
Hi all!

This PR resolves #81 

Kr,

Rshad

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Chef v3.11.1_7.5.1
+
+### Added
+
+- Update to Wazuh version 3.11.1_7.5.1
+
```
Comment (jm404) [inline CHANGELOG.md:15]: Please Add merged PR's to the Changelog:

- #78 
- #80 
- #74 

image

I recently merged #74, please make sure the changes are present in the release branch

**Review by jm404 — CHANGES_REQUESTED** [review]
@rshad Please have a look at requested changes

File: CHANGELOG.md (hunk 1 above)
Comment (rshad) [reply CHANGELOG.md:15]: Hi @jm404 !

CHANGELOG.md is now updated.  f19f6eb

Kr,

Rshad

**Review by jm404 — APPROVED** [review]

**Review by jm404 — APPROVED** [review]
LGTM

**Pull request merged**